python/lsst/ts/m2gui/controltab/tab_alarm_warn.py

```diff
@@ -135,6 +148,63 @@ def _set_text_error_cause(self):
 
         return text_error_cause
 
+    def _set_indicators_limit_switch(self, limit_switch_type):
+        """Set the indicators of limit switch.
```

Comment:
This method doesn't set anything. As I understand it, it creates push buttons serving as indicators. Renaming it to something like "get_indicator_limit_switches" or even better "create_indicators_limit_switch", with the comment "Creates indicators for a limit switch"?

The other option would be to move limit switch indicators to a custom widget class. This will enable you to couple the logic with _get_layout_limit_switch_specific_ring function to create its content (PushButtons for indicators), make tab_alarm_warn widget a bit smaller and enable you to use limit switch indicators in other tabs/widgets as needed.

Reply:
Updated in the commit: 0c06a5b. Thanks!